src/io/perun/meta.clj

```diff
@@ -1,41 +1,75 @@
 (ns io.perun.meta
   "Utilies for dealing with perun metadata"
-  (:require [boot.core :as boot]))
+  (:require [boot.core :as boot]
+            [clojure.java.io :as io]
+            [clojure.string :as string]
+            [io.perun.core :as perun]))
 
 (def +meta-key+ :io.perun)
 
+(def +global-meta-key+ :io.perun.global)
+
+(def +global-meta-defaults+
+  {:doc-root "public"})
```

Comment:
Could you explain your reasoning on the advantages on this?  It's not immediately clear to me.